src/themes/OLH/templates/elements/journal/issue_list_by_decade.html

```diff
@@ -0,0 +1,24 @@
+<div class="large-12 columns">
+    <p>
+        {% blocktrans %}
+            Issues are grouped by decade. Select a decade to view the issues
+            published in during that decade.
+        {% endblocktrans %}
+    </p>
+    <ul class="inline-list no-margin-left">
+        {% for decade, issues in issues_by_decade.items %}
+            <li class="decade_list_item">
+                <a href="#decade-{{ decade }}">{{ decade }}s</a>
+            </li>
+        {% endfor %}
+    </ul>
+</div>
+
+{% for decade, issues in issues_by_decade.items %}
+    <div class="large-12 columns">
+        <h2 id="decade-{{ decade }}">{{ decade }}s</h2>
```

Comment:
I'm not sure this is the best way to form the name of the decade. It looks weird to me with the serif font on the OLH theme. Also if this is public facing shouldn't it be translatable if it has the "s"? 

What do you think about 1990-1999 or just the year, 1990?

Same comment for the clean and material templates.

Reply:
date range wouldn't require localisation, so that would make our life easier